tests/test_formatter.py

```diff
@@ -1,20 +1,21 @@
 from datetime import datetime, timedelta
 from random import randrange
+
 from todoman import ui
 
 DATE_FORMAT = "%d-%m-%y"
 
+
 def test_format_date():
     """
     Tests the format_date function in todoman.ui.TodoFormatter
     """
     formatter = ui.TodoFormatter(DATE_FORMAT)
-    today = datetime.now().date()
+    date_width = formatter.date_width
+    today = datetime.now()
     tomorrow = today + timedelta(days = 1)
-    any_day = today + datetime.timedelta(days = random.randrange(2))
-
-    str_format = lambda x: x.rjust((formatter.date_width), " ")
+    any_day = today + timedelta(days = randrange(2, 8))
 
-    assert formatter.format_date(today) == str_format("Today")
-    assert formatter.format_date(tomorrow) == str_format("Tomorrow")
-    assert formatter.format_date(any_day) == any_day.strftime(DATE_FORMAT)
\ No newline at end of file
+    assert formatter.format_date(today) == "Today".rjust(date_width)
```

Comment:
It's better use `'   Today'` here.  That way, if somebody breaks how `date_width` is calculated in future, this test fill fail and let us know about it immediately.